frontend/apps/web/cypress/support/index.js

```diff
@@ -15,6 +15,9 @@
 
 // Import commands.js using ES2015 syntax:
 import './commands'
-
+Cypress.on('uncaught:exception', (err, runnable) => {
```

Comment:
I think we need uncaught exception, because if there is no element required, test needs to break.